pkgs/servers/irc/charybdis/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchgit, bison, flex, openssl }:
+
+stdenv.mkDerivation rec {
+  name = "charybdis-3.5.0-rc1";
+
+  src = fetchgit {
+    url = "https://github.com/atheme/charybdis.git";
+    rev = "43a9b61c427cd0f3fa2c192890b8a48d9ea6fb7f";
+    sha256 = "ae2c8a72e6a29c901f9b51759b542ee12c4ec918050a2d9d65e5635077a0fcef";
+  };
+
+  patches = [
+     ./remove-setenv.patch
+  ];
+
+  configureFlags = [
+    "--enable-epoll"
+    "--enable-ipv6"
+    "--enable-openssl=${openssl}"
+    "--with-program-prefix=charybdis-"
+  ];
+
+  buildInputs = [ bison flex openssl ];
+
+  meta = {
+    description = "an extremely scalable ircd with some cooperation with the ratbox and ircu guys";
```

Comment:
No dot, according to [the nixpkgs manual](http://nixos.org/nixpkgs/manual/#idm140737319009616):

> description
> A short (one-line) description of the package. This is shown by nix-env -q --description and also on the Nixpkgs release pages.
> Don’t include a period at the end. Don’t include newline characters. Capitalise the first character. For brevity, don’t repeat the name of package — just describe what it does.
> Wrong: "libpng is a library that allows you to decode PNG images."
> Right: "A library for decoding PNG images"